docs/blog/2022-community-update.rst

```diff
@@ -0,0 +1,125 @@
+.. post:: Aug 22, 2022
+   :tags: stats, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - August 2022
+=============================================
+
+Hey there documentarians!
+
+Eric Holscher here, one of the co-founders of Write the Docs.
+Filling in again for the newsletter team in August this year,
+we have a few updates from the community and what has been going on.
+Thanks again to `Aaron <https://www.aaronrcollier.com/>`_ for taking over the newsletter editor role,
+and to `Beth <https://bethaitman.com/>`_ for doing it for the last few years.
+
+We're looking forward to seeing everyone online for our virtual `Prague <http://www.writethedocs.org/conf/prague/2022/>`_ on **September 11-13** in the CEST timezone.
+We're also getting excited about our upcoming virtual `Australia & India <http://www.writethedocs.org/conf/australia/2022/>`_ conference on **December 8-9** in the AEDT timezone.
+The talks were just announced for the Prague conference, and the `Australia conference CFP <https://www.writethedocs.org/conf/australia/2022/cfp/>`_ **is open until September 16 if you have ideas to present.**
+
+We also had our third virtual Portland conference this year.
+It was another success with almost 600 people attending from all over the world.
+The `videos <https://www.youtube.com/playlist?list=PLZAeFn6dfHpnDhFvXG8GprqlLlzSQRBui>`_ from the conference are available online.
+
+2023 Conferences
+----------------
+
+We are looking forward to the rest of our 2022 conferences,
+but we're also starting to plan for 2023.
+We had hoped to do a hybrid event last year,
+but found the logistics quite difficult to figure out.
+You can read more in our `2022 virtual announcement <https://www.writethedocs.org/conf/portland/2022/news/conference-going-virtual/>`_ about the difficulties we had with hybrid.
+
+We know the community is slowly becoming more excited for in-person events as time goes on.
+The hope is to have events in 2023 that are valuable for both our in-person and virtual attendees,
+but we haven't figured out exactly what that looks like.
+
+We continue to work to balance the bandwidth of our organizing team,
+the accessibility that virtual provides,
+and the value from live human interaction that we all miss.
+We will have more announcements about our 2023 plans closer to the end of the year,
+but we appreciate all the support we've gotten from you all as we figure things out.
+
```

Comment:
Do we want to promote the new team org structure? We published it already but it can't hurt to socialize the opportunities to join teams or move around between teams...